src/main/java/com/questhelper/panel/QuestRequirementsPanel.java

```diff
@@ -297,6 +296,10 @@ else if (newColor == Color.ORANGE)
 			{
 				label.setToolTipText("On steel key ring");
 			}
+			else if (newColor == Color.YELLOW)
```

Comment:
Double check this color with https://github.com/Zoinkwiz/quest-helper/pull/1974/files#diff-3307ddf0bbb81c92f7e72c390d7b68ff6632e60154e32453f26a67ca0b877ecbR569